src/cli/subcommands/archive_cmd.rs

```diff
@@ -79,6 +79,11 @@ pub enum ArchiveCommands {
         #[arg(short)]
         depth: Option<ChainEpochDelta>,
     },
+    /// Print block headers at 30 day interval for a snapshot file
```

Comment:
Does it make sense to parametrize the interval?

Reply:
The exact interval matters a little. A week or a quarter would also work fine. Probably not worth it to parameterize.